modules/mobile/app/services/mobile/v0/disability_rating/proxy.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'common/exceptions'
+
+module Mobile
+  module V0
+    module DisabilityRating
+      class Proxy
+        def initialize(user)
+          @user = user
+        end
+
+        def for_user
```

Comment:
`for_user` is a naming convention left over from when we returned an instance of a class with a local user reference. Could this describe the action taken? Maybe something like `.get_disability_ratings` or `.fetch_disability_ratings`.